### ISSUE TITLE
Version detected is incorrect

### ISSUE THREAD

**Issue by cgsmith** [wpscanteam/wpscan#983]
We are running fairly new WordPress (4.6-beta4-38147) and WPScan detects it as 3.1.2 from stylesheets numbers.

This is producing a lot of false positives in our scans.

**Comment by ethicalhack3r** [comment]
We use a number of different techniques to detect the WordPress version. The main fallback which we call 'advanced fingerprinting' uses hashes of client side files, css, js, etc. And we only add these to released versions. We add these hashes manually with every release and unfortunatly we can't do this for every beta release too, due to the time it takes and that not a lot of people (apart from devs) actually install the beta versions on live environments.

TLDR: Detecting beta versions is likely to produce false positives.

**Comment by cgsmith** [comment]
Thanks!
